### ISSUE TITLE
Grabber is flickerung

### ISSUE THREAD

**Issue by Lyr3x** [hyperion-project/hyperion.ng#512]
My Setup:
209 x APA102 
RPI Zero with level shifter

I have some flickering issues mostly on pictures (static) with sand colors or similar. I just found out that there is no problem with the leds but the grabber itself (using this thing without problems for years now). 

When checking the Live TV feature i see how the picture flickers for example between a sand green yelow mix and a more yellow red mix. 

Here is my config:

[Hyperion-2.0.0-Backup (stripes) 2018.07.01.json.zip](https://github.com/hyperion-project/hyperion.ng/files/2152742/Hyperion-2.0.0-Backup.stripes.2018.07.01.json.zip)

**Comment by Lyr3x** [comment]
Yep all wires are fine and the are no voltrage drops or sth else. I changed the grabber to PAL which is a lot better now but there is still flickering. I tried to make a short clip where you can see what i exactly mean
https://photos.app.goo.gl/qXYuPXcZTdsxja3b6
Maybe it is not the grabber itself but the HDMI/CVBS converter which give this flicker signal

**Comment by Phil1988** [comment]
OK first: To me it looks like your setup is different then you wrote or you missed a lot :D

So you are using a HDMI->CVBS converter and a utv007, fushicai, STK1160 or somilar to make the analog video signal digital again (to USB) - so far correct?
If so, then there is nothing you can do to make this more stable.
The "USB Grabber" is not that accurate with making the signal digital (mostly because of low resolution and bad image processing).
There is only one way left to make this super accurate: use a decrypted HDMI signal to calculate its colors - but that of course will cost 4 times the budget of a raspberry and a lot of knowledge to implement this :)

So in short: This behavior is normal.

**Comment by Lyr3x** [comment]
Yeah i already thought that there is just this limitation. It was just not visible with less leds. 

Do you know about differences between those grabbers? I'm using a fushicai grabber and i red about huge picture quality differences in forums.

**Comment by Phil1988** [comment]
I have both - the fushicai is worse but "plug&play" and more stable to the most.

The STK1160 has better color (not oversaturated) and is harder to get working.

I have both and just using the STk1160 because the mixed colors with low satuartion (skin tone, sand ... every "light color") has a better look at this.
But there is a huge community claims since wheezy that this grabber dont work stable.

But I cant install hyperion on my odroid C2 right now :(

**Comment by Phil1988** [comment]
If you have any hints to my problem that would be hightly appreciated

**Comment by Lyr3x** [comment]
What is your problem exactly? I'll be glad to help

**Comment by Phil1988** [comment]
I cant install hyperion.ng on my odroid C2 running ubuntu 16.04 64bit.
My skills with linux are very limited and I dont know what else I can do now.

See here: https://github.com/hyperion-project/hyperion.ng/issues/511

**Comment by Lyr3x** [comment]
So do you get an Error Log while compiling?

Got the stk1160 Yesterday but i dont get any Pictures. Just the small frame warning and Buffer overflow messages...

**Comment by RickDB** [comment]
Have seen the STK1160 defective a lot in the forums over the years (mostly with frame errors), not sure if it's due to clones or just poor overall but if you can get this one:

http://www.insaneware.de/epages/61714203.sf/de_DE/?ObjectPath=/Shops/61714203/Products/usbgrab

**Comment by Lyr3x** [comment]
I tried this one: https://de.aliexpress.com/item/Nsendato-UTV007-USB-2-0-To-HDMI-Video-Catpure-Card-USB2-0-HD-1-Way-Video/32828134783.html

which is way better but gets fairly hot and it has some problems when switchen the input of my avr. But overall the image quality is a lot better resulting in less flickering.

Are there any new high res video grabber?

**Comment by Phil1988** [comment]
@Lyr3x : is it possible that you make a screencapture picture of it to show the coloring?
Would be interessting how much the color losses are.

If yes, please use this test image 
https://i.imgur.com/cQ0TTan.jpg

and grap it with something like this:
`sudo hyperion-v4l2 -d /dev/video0 -f 2 -s 1 --screenshot`

**Comment by tpmodding** [comment]
known issue, discussion is here

https://hyperion-project.org/threads/die-grenzen-eines-analogen-grabbers-und-die-digitale-l%C3%B6sung.4438/

**Comment by Nears** [comment]
> 
> 
> I have both - the fushicai is worse but "plug&play" and more stable to the most.
> 
> The STK1160 has better color (not oversaturated) and is harder to get working.
> 
> I have both and just using the STk1160 because the mixed colors with low satuartion (skin tone, sand ... every "light color") has a better look at this.
> But there is a huge community claims since wheezy that this grabber dont work stable.
> 
> Check this out: https://hyperion-project.org/threads/comparison-fushicai-utv007-vs-stk1160.194/
> 
> But I cant install hyperion on my odroid C2 right now :(

On which system are you running the STk1160? How did you get it running? I am struggeling to set it up on a pi4 with buster :-(  .. no success yet, neither with the old hyperion nor with the hyperion.ng

**Comment by Phil1988** [comment]
I was using a Raspberry Pi 2 B and 3 B.

Im not sure any more. As I am not a programmer it was maybe just a lucky hit :D
It was definetly on the "old" hyperion (not the NG).

**Comment by mazafra1** [comment]
> > I have both - the fushicai is worse but "plug&play" and more stable to the most.
> > The STK1160 has better color (not oversaturated) and is harder to get working.
> > I have both and just using the STk1160 because the mixed colors with low satuartion (skin tone, sand ... every "light color") has a better look at this.
> > But there is a huge community claims since wheezy that this grabber dont work stable.
> > Check this out: https://hyperion-project.org/threads/comparison-fushicai-utv007-vs-stk1160.194/
> > But I cant install hyperion on my odroid C2 right now :(
> 
> On which system are you running the STk1160? How did you (@Phil1988) get it running? I am struggeling to set it up on a pi4 with buster :-( .. no success yet, neither with the old hyperion nor with the hyperion.ng

Me too, but UTV007 + rpi4 and buster. Any luck so far?

**Comment by Nears** [comment]
Not until now 👎 ... still looking into it
